testing_platform/fe/Frontend.Presentation/Pages/Components/TestPreview.razor

```diff
@@ -0,0 +1,78 @@
+﻿@using Contracts.Internal
+
+<RadzenCard Style="width: 100%; border-radius: 0; padding:3%;background-color: transparent">
+    
+    @if (Test.IsPremium)
+    {
+        <div class="premium-type-box">Premium</div>
+    }
+    else
+    {
+        <div class="free-type-box">Free</div>
+    }
+    
+    <RadzenText Text="@Test.Caption" Style="text-align: start;color: #000000;margin: 10px 0;font-size: 26px;font-weight: 400;"/>
+    
+    <RadzenStack Orientation="Orientation.Horizontal" Gap="10px" Style="margin: 0px 0px 50px 0px">
+        <RadzenText Text="@Test.UploadDate.ToString("dd-yyyy")"/>
+        <RadzenText Text="|" />
+        <RadzenText Text="@($"{Test.DurationInMinutes.ToString()} min")"/>
+    </RadzenStack>
+    
+    <RadzenStack Orientation="Orientation.Horizontal" Gap="10px">
+        <RadzenText Style="font-weight: 600" Text="Category: " />
+        <RadzenText Style="font-weight: 600" Text="@Test.Category" />
+    </RadzenStack>
+    
+    <RadzenStack Orientation="Orientation.Horizontal" Gap="10px">
+        <RadzenText Style="font-weight: 600" Text="Complexity: " />
+        <RadzenText Style="font-weight: 600" Text="@Test.Complexity" />
+    </RadzenStack>
+    
+    <div class="tags-wrapper">
+        @foreach (var tag in Test.Tags)
+        {
+            <RadzenBadge BadgeStyle="BadgeStyle.Primary" Text="@tag" />
+        }
+    </div>
+    
+    <RadzenStack Orientation="Orientation.Horizontal" JustifyContent="JustifyContent.SpaceBetween">
+        
+        @if (Test.IsCompleted)
```

Comment:
Simplify the code by using a ternary operator

Reply:
simplified by string.Concat